### ISSUE TITLE
getModel() bug when id equals 0

### ISSUE THREAD

**Issue by jgreener64** [3dmol/3Dmol.js#270]
In the `GLViewer.getModel()` function:

```
this.getModel = function(id) {
  id = id || models.length - 1;
  return models[id];
};
```

if id equals 0 the last model will be returned because 0 cannot satisfy the OR condition. So `getModel(0)` will return the last model added, not the first model.

Is this a bug or am I missing something? Thanks very much.

**Comment by dkoes** [comment]
Nope, that's a bug.  Fixed 0f2de49

**Comment by jgreener64** [comment]
Thanks
